astracliGettingStarted/README.md

```diff
@@ -0,0 +1,342 @@
+# Overview
+In this guide we will learn how to use the DataStax Astra command line interface (Astra CLI) with both Astra DB and Astra Streaming. The Astra CLI is a set of commands used to create and manage your Astra resources. The Astra CLI is designed to get you working quickly with Astra, with an emphasis on automation. The CLI enables you to work with your Astra DB resources, query data through CQLSH, load data with DSBulk, configure and manage your Astra Streaming artifacts, and define Users and Roles.
```

Comment:
Potentially use the contraction "we'll" instead of "we will" to be more in line with the style used in other guides.

Reply:
It might be cool at add an image of the CLI?